src/legacy/core_plugins/vis_type_timelion/public/components/timelion_expression_input_helpers.ts

```diff
@@ -18,18 +18,17 @@
  */
 
 import { get, startsWith } from 'lodash';
-import PEG from 'pegjs';
 import * as monacoEditor from 'monaco-editor/esm/vs/editor/editor.api';
 
+import { i18n } from '@kbn/i18n';
+import { Parser } from 'pegjs';
+
 // @ts-ignore
-import grammar from 'raw-loader!../chain.peg';
+import { parse } from '../chain';
```

Comment:
I prefer to mark somehow that this file was automatically generated from PEG, for example. for `kuery.peg` we moved it to the _`generated_` folder